src/parser/jobs/blm/Reopeners.js

```diff
@@ -0,0 +1,108 @@
+import React, {Fragment} from 'react' // eslint-disable-line
```

Comment:
This entire file is a nope as-is. Never disable `eslint` without _good_ reason, and if you have one, explain it in a comment.

If this file is WIP, it shouldn't be in a PR without a WIP tag.